code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -340,8 +285,8 @@
 	initial_gas_mix = "TEMP=180"
 	slowdown = 2
 	environment_type = "snow"
-	sand_type = /obj/item/stack/sheet/mineral/snow
 	flags_1 = NONE
+	archdrops= list(/obj/item/stack/sheet/mineral/snow = 5)
```

Comment:
Should put another space after `archdrops` just for consistency.

Reply:
i hate you